setup.py

```diff
@@ -11,12 +11,12 @@
     classifiers=["Programming Language :: Python :: 3 :: Only"],
     py_modules=["tap_bingads"],
     install_requires=[
-        "arrow==0.12.0",
-        "bingads==13.0.1",
-        "requests==2.20.0",
+        "arrow==0.15.5",
+        "bingads==13.0.2",
+        "requests==2.23.0",
         "singer-python==5.9.0",
         "stringcase==1.2.0",
-        "backoff==1.8.0",
+        "backoff==1.10.0",
```

Comment:
did you update these things for a reason or just because? It's not a bad thing to do, just be mindfull that whenever you are upgrading packages, you need to be carefull because there's a chance it will change its API. I try to not do it if there's no immediate need I have that is meet by a newer package, or if there is not a security concern that is addressed by a newer version.

Reply:
I did it for no reason. I will not change it for no reasons next time